<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UnTech::GuiQt::MetaSprite::ExportOrderEditorWidget</class>
 <widget class="QWidget" name="UnTech::GuiQt::MetaSprite::ExportOrderEditorWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>480</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTreeView" name="treeView">
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolBar" name="toolBar">
     <property name="movable">
      <bool>false</bool>
     </property>
     <property name="iconSize">
      <size>
       <width>16</width>
       <height>16</height>
      </size>
     </property>
     <property name="floatable">
      <bool>false</bool>
     </property>
     <addaction name="action_AddFrame"/>
     <addaction name="action_AddAnimation"/>
     <addaction name="action_AddAlternative"/>
     <addaction name="action_CloneSelected"/>
     <addaction name="action_RemoveSelected"/>
    </widget>
   </item>
  </layout>
  <action name="action_AddFrame">
   <property name="icon">
    <iconset resource="../../../../resources/resources.qrc">
     <normaloff>:/icons/add.svg</normaloff>:/icons/add.svg</iconset>
   </property>
   <property name="text">
    <string>Add Frame</string>
   </property>
  </action>
  <action name="action_AddAnimation">
   <property name="icon">
    <iconset resource="../../../../resources/resources.qrc">
     <normaloff>:/icons/add.svg</normaloff>:/icons/add.svg</iconset>
   </property>
   <property name="text">
    <string>Add Animation</string>
   </property>
  </action>
  <action name="action_AddAlternative">
   <property name="icon">
    <iconset resource="../../../../resources/resources.qrc">
     <normaloff>:/icons/add.svg</normaloff>:/icons/add.svg</iconset>
   </property>
   <property name="text">
    <string>Add Alternative</string>
   </property>
  </action>
  <action name="action_CloneSelected">
   <property name="icon">
    <iconset resource="../../../../resources/resources.qrc">
     <normaloff>:/icons/clone.svg</normaloff>:/icons/clone.svg</iconset>
   </property>
   <property name="text">
    <string>Clone Selected</string>
   </property>
   <property name="toolTip">
    <string>Clone Selected</string>
   </property>
  </action>
  <action name="action_RemoveSelected">
   <property name="icon">
    <iconset resource="../../../../resources/resources.qrc">
     <normaloff>:/icons/remove.svg</normaloff>:/icons/remove.svg</iconset>
   </property>
   <property name="text">
    <string>Remove Selected</string>
   </property>
   <property name="toolTip">
    <string>Remove Selected</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../../../../resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
